jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamInvokeRunner.kt

```diff
@@ -43,7 +44,7 @@ class SamInvokeRunner : AsyncProgramRunner<RunnerSettings>() {
         // Requires SamDebugSupport too
         if (DefaultDebugExecutor.EXECUTOR_ID == executorId) {
             profile.settings.runtime?.let {
-                return SamDebugSupport.supportedRuntimeGroups.contains(Runtime.fromValue(it).runtimeGroup)
+                return SamDebugSupport.supportedRuntimeGroups.contains(Runtime.fromValue(it).nullable?.runtimeGroup)
```

Comment:
I dont think we actually need this, since UNKNOWN_TO_SDK will lead to null runtime group anyway